pyproject.toml

```
@@ -0,0 +1,43 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [ "setuptools>=45", "setuptools_scm[toml]>=6.2",]
+
+[project]
+classifiers = [ "Development Status :: 2 - Pre-Alpha", "Natural Language :: English", "Programming Language :: Python :: 3",]
+description = "Happi Database Access for LCLS Beamline Devices"
+dynamic = [ "version", "readme", "dependencies", "optional-dependencies", "optional-dependencies",]
+keywords = []
+name = "happi"
+requires-python = ">=3.9"
+[[project.authors]]
+name = "SLAC National Accelerator Laboratory"
+
+[options]
+zip_safe = false
+include_package_data = true
+
+[project.license]
+file = "LICENSE.md"
+
+[project.scripts]
+happi = "happi.cli:happi_cli"
+
+[tool.setuptools_scm]
+write_to = "happi/_version.py"
+
+[tool.flake8]
+exclude = "docs/*,versioneer.py"
+
+[tool.setuptools.packages.find]
+where = [ ".",]
+include = [ "happi",]
```

Looks like this is still generated with a previous version of the migration tools; could you take a look at your checked out copy?
```suggestion
include = [ "happi*",]
```

---

argh I swear I pulled before running this

---

fixed in https://github.com/pcdshub/pcds-python-migration-tools/pull/6 👍